src/settings/settings.ts

```diff
@@ -0,0 +1,52 @@
+import { invoke } from '@tauri-apps/api';
+import { SettingsManager } from 'tauri-settings';
+
+interface Schema {
+  project: {
+    name: string;
+  };
+  openAI: {
+    apiKey: string;
+    completeModel: string;
+    chatModel: string;
+  };
+}
+
+const settingsManager = new SettingsManager<Schema>(
+  {
+    project: {
+      name: 'project-x',
+    },
+    openAI: {
+      apiKey: await readEnv('OPENAI_API_KEY'),
+      completeModel: await readEnv('OPENAI_COMPLETE_MODEL'),
+      chatModel: await readEnv('OPENAI_CHAT_MODEL'),
+    },
+  },
+  {
+    fileName: 'settings',
```

Comment:
Does this persist settings in `settings.json`?

Reply:
Yes, the lib appends the extension and ignores the extension if we define it.
I've also made the `dir` explicit to help discover where the file is.